_benchmark/quickstart.md

```diff
@@ -0,0 +1,421 @@
+---
+layout: default
+title: Quickstart
+nav_order: 2
+---
+
+# OpenSearch Benchmark quickstart
+
+This tutorial outlines how to quickly install OpenSearch Benchmark (OSB) and run your first OpenSearch Benchmark workload. 
+
+## Prerequisites
+
+To perform the Quickstart steps, you'll need to fulfill the following prerequisites:
+
+- A currently active OpenSearch cluster. For instructions on how to create an OpenSearch cluster, see [Creating a cluster]({{site.url}}{{site.baseurl}}//tuning-your-cluster/index/).
+- Git 2.3 or greater.
+
+Additional prerequisites are required depending on your installation method.
+
+- If you plan to install OpenSearch Benchmark with [PyPi](https://pypi.org/), install Python 3.8 or greater.
+- If you plan to install OpenSearch Benchmark using Docker, install Docker. 
+
+## Installing OpenSearch Benchmark
+
+You can install OpenSearch Benchmark either using PyPi or Docker. 
+
+If you plan to run OpenSearch Benchmark with a cluster using Sigv4, see [Sigv4 support]({{site.url}}{{site.baseurl}}/benchmark/tutorials/sigv4)
+
+### PyPi
+
+To install OpenSearch Benchmark with PyPi, enter the following `pip` command:
+
+```bash
+pip3 install opensearch-benchmark
+```
+{% include copy.html %}
+
+After the installation completes, verify that OSB is running by entering the following command:
+
+```bash
+opensearch-benchmark --help
+```
+{% include copy.html %}
+
+If successful, OpenSearch returns the following response:
+
+```bash
+$ opensearch-benchmark % opensearch-benchmark --help
```

Comment:
`$ opensearch-benchmark % opensearch-benchmark` --> We can remove `opensearch-benchmark %`